projects/uitdatabank/reference/entry.json

```diff
@@ -9496,6 +9496,320 @@
           }
         ]
       }
+    },
+    "/cultuurkuur/regions": {
+      "get": {
+        "summary": "/cultuurkuur/regions",
+        "tags": ["Cultuurkuur"],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "headers": {
+              "content-type": {
+                "schema": {
+                  "type": "string",
+                  "example": "application/json"
+                },
+                "description": "application/json"
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "label": {
+                        "type": "string",
+                        "description": "Name region"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "NIS code"
+                      },
+                      "children": {
```

Comment:
We should think about more expressive names, for example `regions` and `subregions`

Reply:
I am open to that, but maybe we should have that discussion with Emma and then change it in a later PR. I think the current way might it easier to write some recursive code in JS, but that is just speculation.